appveyor.yml

```diff
@@ -89,12 +89,13 @@ environment:
     CI_JOB_NAME: dist-x86_64-msvc
   - RUST_CONFIGURE_ARGS: >
       --build=i686-pc-windows-msvc
-      --target=i586-pc-windows-msvc
+      --target=i586-pc-windows-msvc,aarch64-pc-windows-msvc
       --enable-full-tools
       --enable-profiler
     SCRIPT: python x.py dist
     DEPLOY: 1
     CI_JOB_NAME: dist-i686-msvc
+    APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017 Preview
```

Comment:
Ok AppVeyor [got back to us](https://help.appveyor.com/discussions/questions/27382-arm64-support-in-the-default-image#comment_46383941), and it looks like this may want to drop the "Preview" and we can see if that works?

Reply:
I was specifically using the Preview image because I had been using VS 2017 Preview to compile Rust locally and the [AppVeyor Windows software page](https://www.appveyor.com/docs/windows-images-software/#visual-studio-2017) made it sound like there was a VS 2017 Preview image for this purpose.  (Microsoft's release notes also make it sound like there have been various fixes for AArch64 codegen and whatnot in the preview toolchains.)  Can we confirm with AppVeyor that there is an image type for accessing the Preview toolchains and what the correct name to use is?

In parallel, I guess we could switch over to the standard image(s) and see if that improved the situation?

Reply:
Sure yeah, I will ask.